directory/datadog_checks/directory/config.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, instance):
         self.filegauges = is_affirmative(instance.get('filegauges', False))
         self.countonly = is_affirmative(instance.get('countonly', False))
         self.ignore_missing = is_affirmative(instance.get('ignore_missing', False))
+        self.report_missing = is_affirmative(instance.get('report_missing', False))
```

Comment:
Let's document in config too

Reply:
This is now removed, forgot to remove it from here